core/src/main/scala/akka/persistence/dynamodb/internal/JournalDao.scala

```diff
@@ -58,6 +63,34 @@ import software.amazon.awssdk.services.dynamodb.model.Update
 
   private implicit val ec: ExecutionContext = system.executionContext
 
+  private val dateHeaderLogCounter = new AtomicLong
+  private val dateHeaderParser = DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss z", Locale.US)
+
+  private def checkClockSkew(response: SdkResponse): Unit = {
+    try {
+      if (dateHeaderLogCounter.getAndIncrement() % 1000 == 0) {
+        val dateHeaderOpt = response.sdkHttpResponse().firstMatchingHeader("Date")
+        if (dateHeaderOpt.isPresent) {
+          val dateHeader = dateHeaderOpt.get
+          val awsInstant = Instant.from(dateHeaderParser.parse(dateHeader))
+          val now = Instant.now()
+          // The Date header only has precision of seconds so this is just a rough check
+          if (math.abs(java.time.Duration.between(awsInstant, now).toMillis) >= 2000) {
```

Comment:
Make the tolerance configurable?

Reply:
Yes, also good to be able to disable that way:https://github.com/akka/akka-persistence-dynamodb/pull/113/commits/c6a46b526d08920aa3cb3a36cf6ed634b2669e36

Reply:
👍🏼